app/containers/message/Card.js

```diff
@@ -55,7 +55,7 @@ export default class Cards extends React.PureComponent {
 					</Card>
 				</TouchableOpacity>
 				<PhotoModal
-					title={this.props.data.title}
+					title={this.props.data.title || 'test'}
```

Comment:
🤔

Reply:
Just a workaround to fix PropTypes.string.isRequired.
It will be fixed later on #55 .